### DISCUSSION TITLE
Getting latest beta for docker?

### DISCUSSION THREAD

**Discussion by lonevvolf** [steilerDev/icloud-photos-sync#332] [Q&A]
Sorry if I'm missing something here, but how can we get the newest beta release ([v1.2.0-beta.4](https://github.com/steilerDev/icloud-photos-sync/releases/tag/v1.2.0-beta.4) at the moment) from DockerHub?  I see that the image has been updated there fairly recently, but when I get the latest version in my Docker installation, I am still seeing v1.1.1-beta.3 in the logs.  Is this something I am doing wrong, or is the actual version reported in the container incorrect?

**Comment by steilerDev** [comment]
This is weird - let me check later this weekend, planning to release the latest nightly to main anyway, which should resolve the issue.

**Comment by steilerDev** [comment]
Confirmed that [beta.5](https://github.com/steilerDev/icloud-photos-sync/releases/tag/v1.2.0-beta.5) is available from Dockerhub and is reporting the correct version within the app. Planning to release this to prod this week.

**Comment by lonevvolf** [comment]
Strange - I'm using the Container Manager on Synology, and it usually tells me when new versions are available.  But this one doesn't pop up with a new version.  I guess I'll wait for the prod version.

**Comment by lonevvolf** [comment]
Figured it out - have to "Reset" the container to have it pick up changes to the image.  Not sure why it doesn't alert when there are changes to the image in the Docker Hub, but at least I can force update it.
